### PR TITLE
Add fuzzing for PE/COFF unwind data parsers in libunwindstack

### PR DESCRIPTION
**Pull request by ronaldfw**
We already have a fuzzer for the main PE/COFF parsing part, which is
done in PeCoffInterface, but there is more data that we need to parse
from PE/COFF files, namely the unwind infos (UNWIND_INFO struct) and
runtime functions (RUNTIME_FUNCTIONS struct). This change adds fuzzers
for both parsers. Fuzzers are basic without any structure awareness.

Tested: Built and ran fuzzers with local oss-fuzz clone.
Bug: http://b/194768602

### CODE REVIEW

File: third_party/libunwindstack/tests/fuzz/PeCoffRuntimeFunctionsFuzzer.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <inttypes.h>
+#include <cstddef>
+
+#include <unwindstack/Memory.h>
+#include <unwindstack/PeCoffInterface.h>
+#include "../../PeCoffRuntimeFunctions.h"
```
Comment (beckerhe) [inline third_party/libunwindstack/tests/fuzz/PeCoffRuntimeFunctionsFuzzer.cpp:22]: Not sure what's the policy is for Android/libunwindstack but in Orbit we try to avoid include paths with `../` because it's hard to see if someone is trying to circumvent proper module dependencies.

In this case it's fine but we could avoid it by adding another target_include_directories call to the CMakeLists file and add the libunwindstack directory to the include path for the fuzz target. WDYT?

File: third_party/libunwindstack/tests/fuzz/PeCoffUnwindInfosFuzzer.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <inttypes.h>
+#include <cstddef>
+
+#include <unwindstack/Memory.h>
+#include <unwindstack/PeCoffInterface.h>
+#include "../../PeCoffUnwindInfos.h"
```
Comment (beckerhe) [inline third_party/libunwindstack/tests/fuzz/PeCoffUnwindInfosFuzzer.cpp:22]: Same as above

File: third_party/libunwindstack/tests/PeCoffEpilogTest.cpp [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
 #include "PeCoffEpilog.h"
 
+#include <array>
```
Comment (beckerhe) [inline third_party/libunwindstack/tests/PeCoffEpilogTest.cpp:19]: Was this change intended? (I quickly skimmed through the file and didn't see any usage of std::array.)

**Review by beckerhe — APPROVED** [review]
Nice. Otherwise LGTM!

File: third_party/libunwindstack/tests/PeCoffEpilogTest.cpp (hunk 3 above)
Comment (ronaldfw) [reply third_party/libunwindstack/tests/PeCoffEpilogTest.cpp:19]: Yes, this is actually used here, none of our builds complain, but the oss-fuzz build doesn't work on my machine because of this missing include.

Comment (beckerhe) [reply third_party/libunwindstack/tests/PeCoffEpilogTest.cpp:19]: Cool. Thanks for clarifying!

**Comment by beckerhe** [comment]
BTW a fix for the feedback/copybara check is on its way: https://github.com/google/orbit/pull/3352

File: third_party/libunwindstack/tests/fuzz/PeCoffRuntimeFunctionsFuzzer.cpp (hunk 1 above)
Comment (ronaldfw) [reply third_party/libunwindstack/tests/fuzz/PeCoffRuntimeFunctionsFuzzer.cpp:22]: Yeah, there are some cases in libunwindstack tests where "../" is used. But I prefer your suggestion and adjusted the CMakeLists file (let me know if I did the right thing, or otherwise if there is a better way).

Comment (beckerhe) [reply third_party/libunwindstack/tests/fuzz/PeCoffRuntimeFunctionsFuzzer.cpp]: Perfect. Thank you!
